go/vt/vttablet/tabletmanager/vdiff/workflow_differ.go

```diff
@@ -343,9 +342,6 @@ func (wd *workflowDiffer) buildPlan(dbClient binlogplayer.DBClient, filter *binl
 		if len(specifiedTables) != 0 && !stringListContains(specifiedTables, table.Name) {
 			continue
 		}
-		if schema.IsInternalOperationTableName(table.Name) {
```

Comment:
This is the most important code change in this PR. We were ignoring internal tables for vdiff and hence vdiffs on online ddls were failing.